src/errors.ts

```diff
@@ -57,3 +57,15 @@ export class UninitializedPurchasesError extends Error {
         Object.setPrototypeOf(this, UninitializedPurchasesError.prototype);
     }
 }
+
+/**
+ * @internal
+ */
+export class UnsupportedPlatformError extends Error {
+    constructor() {
+        super("This method is not available in the current platform.");
+
+        // Set the prototype explicitly.
+        Object.setPrototypeOf(this, UnsupportedPlatformError.prototype);
```

Comment:
Yeah I don't think it's really necessary in this case.

Reply:
So when I remove this and run tests on the CI machine, tests fail because `instanceof` doesn't seem to work well. Tests do pass correctly in my computer correctly without this though :/

Reply:
![image](https://user-images.githubusercontent.com/3922667/214412551-e10a8dc3-fd99-4de4-ba06-ae59b28ca6bd.png)